### PR TITLE
added activity info

### PR DESCRIPTION
**Pull request by raoulesd**
can be merged once activity e-mail has been approved for sending

### CODE REVIEW

File: content/schedule/activities.md [hunk 1]
```diff
@@ -7,4 +7,38 @@ duration: 3h30m
...
+---
+- Beer Tasting in Local Delft Brewery
+
+You will visit the local Delft brewery 'de Koperen Kat', or, in English, 'the Copper Cat' [(Google Maps link)](https://goo.gl/maps/eXBfGVkCqJzyyMmK9), and you will get to sample a few different beers they make there.
+
+---
+- Other activity
+
+Take matters into your own hands and embark on an adventure in either Delft, Rotterdam or the Hague! You are free to do what you like in the time of the second round.
+We provide an overview of suggestions for other activities [on this map](https://wisv.ch/map). Note that all 'other activities' are self-paid and on your own accord. We do **not** reimburse any travel/entrance cost of this.
```
Comment (verwoerd) [inline content/schedule/activities.md:41]: I would add a list with interesting activities to do, like the city tours, musea (vermeer centrum, prinsenhof, botanical garden, etc.)

Comment (mpsijm) [reply content/schedule/activities.md:41]: This list is hidden in https://wisv.ch/map :slightly_smiling_face:

Comment (raoulesd) [reply content/schedule/activities.md:41]: should I still embed the list in here? or do you guys think referencing the wisv.ch/map is sufficient?

Comment (mpsijm) [inline content/schedule/activities.md:41]: Separate comment: we also do not reimburse travel costs to the Canal/Church/Copper Cat, right? This sentence makes it sound like we do (but of course, we only pay for the entrance fee) :stuck_out_tongue:

Comment (verwoerd) [reply content/schedule/activities.md:41]: I would embed the list here, since the map is 'omslachtig' to discover the options.

Comment (raoulesd) [reply content/schedule/activities.md]: @mpsijm should be more clear now!

Comment (raoulesd) [reply content/schedule/activities.md]: done!

**Review by verwoerd — APPROVED** [review]

File: content/schedule/activities.md (hunk 1 above)
Comment (mpsijm) [reply content/schedule/activities.md]: Yes, thanks! :smile:

**Review by mpsijm — APPROVED** [review]

**Pull request merged**